src/modules/status.rs

```diff
@@ -52,6 +59,55 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     }
 }
 
+#[cfg(not(windows))]
+fn to_text_status(exit_code: &str) -> &str {
+    ""
+}
+
+#[cfg(windows)]
+fn to_text_status(exit_code: &str) -> &str {
+    use winapi::shared::ntdef::NTSTATUS;
+    use winapi::shared::ntstatus::*;
+
+    exit_code
+        .parse::<NTSTATUS>()
+        .ok()
+        .map_or("", |code| match code {
+            STATUS_ACCESS_VIOLATION => " (ACCESS_VIOLATION)",
```

Comment:
I'd prefer if adding the braces and space was handled in the format string.

Reply:
Is there a way to do conditional formating in starship? The issue here is that in most normal cases, the text_status will be empty and I'd like to avoid the " ()" in that common case.

Reply:
Format strings have [conditional formatting](https://starship.rs/config/#conditional-format-strings). Enclosing the variable in braces ensures that it'll only get printed if the varible has a value (the other braces would need to escaped though).

Reply:
Great, thanks for the pointer!. Removed the formatting.